src/quote/provider/index.ts

```diff
@@ -37,12 +41,14 @@ export enum FlashMintContractType {
   leveraged = 1,
   leveragedAerodrome = 2,
   leveragedExtended = 3,
-  nav = 4,
-  wrapped = 5,
-  zeroEx = 6,
+  leveragedMorphoAaveLM = 4,
+  nav = 5,
+  wrapped = 6,
+  zeroEx = 7,
 }
 
 export interface FlashMintQuoteRequest {
+  // TODO: add taker?
```

Comment:
will probably do this when we have the new FM contracts for 0x v2 and LIFI. for now it's not necessary yet.